### ISSUE TITLE
unfave all tweets made in a certain period

### ISSUE THREAD

**Issue by hansvangent** [grote/twitter-unfav#1]
Is it possible to have an option to unfave all tweets made between date X and Y?

**Comment by grote** [comment]
Yes, you would just need to adapt this line: https://github.com/grote/twitter-unfav/blob/master/twitter-unfav.py#L88

**Comment by hansvangent** [comment]
Sorry not really a Python programmer :-/ ideally it would be a config option like ./twitter-unfav.py --startdate=MM-DD-YYY --enddate=MM-DD-YYYY or something like that

**Comment by grote** [comment]
Yes that would be nice, but I'm swamped with other work at the moment and this feature isn't scratching an itch of mine. Sorry.

**Comment by hansvangent** [comment]
No worries, will try to look into it and otherwise do a pull request :-)
